cli/src/main/java/org/jboss/as/cli/Util.java

```diff
@@ -1187,6 +1187,10 @@ public static String getMessagesFromThrowable(Throwable t){
             }
             t1 = t1.getCause();
         }
+
+        for (Throwable suppressed : t.getSuppressed()) {
+            buf.append("\n").append(suppressed.getLocalizedMessage());
```

Comment:
Should this have some sort of explanatory text to indicate that these are suppressed exceptions? We don't do that above for the getCause() handling, but now with 2 possible types of extra information does it get more confusing?

Should this be recursive:?

 buf.append("\n").append(getMessagesFromThrowable(suppressed));

If not should it have the if (t1.getLocalizedMessage() != null) { ... } else {...} handling like we do above? getLocalizedMessage() will be null if no details were provided to the exception constructor.